app/game/game.js

```diff
@@ -117,15 +117,18 @@ const GAME = (humanUsername, aiUsername, gameId) => {
         alertPlayerOfThreat(THREATS_INDEXED_BY_ROOM[room], HUMAN_USERNAME, room);
     }
 
-    const onThreatUnresolved = (room) => {
+    const removeThreat = (room) => {
         delete THREATS_INDEXED_BY_ROOM[room] // Remove threat
         AVAILABLE_ROOMS.splice(AVAILABLE_ROOMS.indexOf(room), 1); // Room no longer available
```

Comment:
Oop, you should swap this line with 127 that starts with `ROOMS_WITH_THREATS.splice...`. You only want to splice the `AVAILABLE_ROOMS` when a threat is unresolved, but you always want to splice `ROOMS_WITH_THREATS`